docs/database/azure-postgresql-entity-framework-integration.md

```diff
@@ -0,0 +1,36 @@
+---
+title: .NET Aspire Azure PostgreSQL Entity Framework Core integration
+description: Learn how to integrate Azure PostgreSQL with .NET Aspire applications, using both hosting and Entity Framework Core client integrations.
+ms.date: 01/21/2025
+uid: dotnet/aspire/azure-postgresql-entity-framework-integration
+---
+
+# .NET Aspire Azure PostgreSQL Entity Framework Core integration
+
+[!INCLUDE [includes-hosting-and-client](../includes/includes-hosting-and-client.md)]
+
+[Azure Database for PostgreSQL](/azure/postgresql/)—Flexible Server is a relational database service based on the open-source Postgres database engine. It's a fully managed database-as-a-service that can handle mission-critical workloads with predictable performance, security, high availability, and dynamic scalability. The .NET Aspire Azure PostgreSQL integration provides a way to connect to existing Azure PostgreSQL databases, or create new instances from .NET with the [`docker.io/library/postgres` container image](https://hub.docker.com/_/postgres).
+
+## Hosting integration
+
+[!INCLUDE [postgresql-flexible-server](includes/postgresql-flexible-server.md)]
+
+### Hosting integration health checks
```

Comment:
This only happens when you use `RunAsContainer` as we get it for free from the local PostgreSQL hosting integration. We don't have a health check when targeting the Azure resource in the Hosting layer. Maybe this section can be dropped?